utils/cloneByDate.py

```diff
@@ -57,10 +55,7 @@
 
 def confirm(txt, options):
     if not options.assumeyes:
-        if six.PY2:
-            inputfn = raw_input
-        else:
-            inputfn = input
+        inputfn = input()
         response = inputfn(txt)
         while ['y', 'n'].count(response.lower()) == 0:
             response = inputfn(txt)
```

Comment:
I think this is not correct. Apparently what you want would be:
```suggestion
        inputfn = input
        response = inputfn(txt)
        while ['y', 'n'].count(response.lower()) == 0:
            response = inputfn(txt)
```


But, since you're dropping the Python 2 compatibility, then it does not make sense anymore to have an `inputfn` alias. Therefore I think we should go like:

```suggestion
        response = input(txt)
        while ['y', 'n'].count(response.lower()) == 0:
            response = input(txt)
```

Reply:
I did not look left or right/up or down....